torch/csrc/api/include/torch/nn/module.h

```diff
@@ -581,11 +581,11 @@ void Module::to_impl(Ts&&... ts) {
   }
   // Then move every parameter to the new dtype/device.
   for (auto& parameter : parameters_) {
-    parameter->set_data(autograd::Variable(*parameter).data().to(ts...));
+    parameter->set_data(autograd::Variable(*parameter).to(ts...));
```

Comment:
It's okay to remove `.data()` here, because `set_data(tensor)` ignores `tensor`'s AutogradMeta, so it doesn't matter whether the `tensor` passed in has AutogradMeta or not.